content/06.discussion-a.md

```diff
@@ -1,82 +1,69 @@
 ## Discussion
 
-### General pargrah on what GitHub can enable in EcoEvo  
-*Contributors to this section: Rob, Brandon*   
+<!-- ### General paragraph on what GitHub can enable in EcoEvo  
+*Contributors to this section: Rob, Brandon*   -->
 
-There have been many calls for researchers outside of the software development community to join the 73 million GitHub users for their collaborative research.  
-This call comes in light of the continual shift toward a more open-science framework across several fields, and as computational and data requirements increase in several fields.
-Until now, resources and practical guidance specifically focused on using GitHub within the EEB community have been dispersed in blog posts and video tutorials.
-We felt these resources have been extremely useful for us to learn to use Github in our own work, and we felt that a collation of the main ideas into one medium, while adding on our personal perspectives, would be of use in the EEB community. 
+There have been many calls for researchers outside of the software development community to leverage GitHub for their collaborative research.  
+These calls come in light of the continual shift toward science that is more open as well as computationally and data intensive.  
+Until now, resources and practical guidance specifically focused on using GitHub within the EEB community have been dispersed in blog posts and video tutorials.  
+While these resources have been extremely useful for learning to use GitHub, we provide a more comprehensive review of the key benefits that GitHub can bring to EEB researchers regardless of career stage or coding experience.  
 
 In this paper, we describe 13 tractable ways that EEB researchers can leverage GitHub to enable more transparent and collaborative research (Figure 2).  
```

Comment:
```suggestion
In this paper, we describe 13 tractable ways that EEB researchers can leverage GitHub to enable more transparent and collaborative research (Figure @fig:scatterblob).  
```
Which figure is this supposed to reference?  If it's `@fig:scatterblob`, then my only hesitation is that there are 15 things on that figure, not 13.

Reply:
Agreed that this will likely stand out so we should figure out what we want to do. I think that there are 15 in the figure because we have subsections for 13 use cases, but then a couple are also captured under the 'additional uses' section. @LunaSare I think we discussed this a bit in one of our meetings about the figure, but wondering what you all think we should do? Remove the citation here? Elaborate somewhere on why this figure has 15 but we have only 13 sub-headings?